src/components/pages/profile-overview/profile-overview.tsx

```diff
@@ -101,27 +99,27 @@ const ProfileOverview: React.FC<{
       </NeutralLink>
       <h2>Addresses</h2>
       <SectionBox>
-        {!data ? (
+        {!userAddresses ? (
           <BoxedLink disableClick={true} href="#">
             <SkeletonTextLine fontSize={1.6} width={50} />
           </BoxedLink>
         ) : (
           <UserAddresses
             hideAddressList={hideAddressList}
-            addressList={data.userAddresses}
+            addressList={userAddresses}
           />
         )}
       </SectionBox>
-      {data && data.userAddresses.length > 1 ? (
+      {userAddresses && userAddresses.length > 1 ? (
         <Flex>
           <ShowMoreButton
             hideList={hideAddressList}
-            quantity={data.userAddresses.length - 1}
+            quantity={userAddresses.length - 1}
             setHideList={setHideAddressList}
           />
         </Flex>
       ) : null}
-      {data ? (
+      {userAddresses ? (
         <NeutralLink href="/account/profile/addresses/new">
           <FakeButton variant={ButtonVariant.SECONDARY}>
             + Add new address
```

Comment:
The button should be always visible, the user should be able to add an address anyway

Reply:
Or we consider that to have any address, the user should have a mandatory `Profile`?
This kinda makes sense don't you think? Because, what if the purpose to have an address if it is not linked to a name? 🧐

Reply:
I asked myself this question yesterday, and I finally decided to display it conditionally to the moment `userAddress` was defined. So until data is ready to be displayed, we have skeleton component for addresses, and I wanted to hide the button during this time, and display it with the data. It's more a "display/UX" concern in fact.

Reply:
We did the same for `identities` buttons if I remember well, just to be sure that a user won't try to click buttons before data arrived client side

Reply:
> the moment userAddress was defined

That's my point, I think it should always be defined, at least as `[]`

But for 

> the user should have a mandatory Profile?

Addresses or not, if s·he don't have a profile, s·he shouldn't be able to create and address, so what I think is that we should change `userAddresses ?` to `userProfile ?` for the `create an address` button

Reply:
> Addresses or not, if s·he don't have a profile, s·he shouldn't be able to create and address, so what I think is that we should change userAddresses ? to userProfile ? for the create an address button

Indeed. But as it is done for now, this isn't an issue because you get redirected to the creation of profile form if you haven't any profile yet, so you never have the chance to see this page and this button 🙂 

But, if we decide to go for a "create profile" button, we should then change that, I agree.